tests/browser_context_options_test.go

```diff
@@ -97,17 +97,16 @@ func TestBrowserContextOptionsExtraHTTPHeaders(t *testing.T) {
 	p := bctx.NewPage()
 
 	err := tb.awaitWithTimeout(time.Second*5, func() error {
-		tb.promiseThen(p.Goto(tb.URL("/get"), nil),
-			func(resp api.Response) {
-				require.NotNil(t, resp)
-				var body struct{ Headers map[string][]string }
-				require.NoError(t, json.Unmarshal(resp.Body().Bytes(), &body))
-				h := body.Headers["Some-Header"]
-				require.NotEmpty(t, h)
-				assert.Equal(t, "Some-Value", h[0])
-			})
+		gotoPage := tb.promise(p.Goto(tb.URL("/get"), nil))
```

Comment:
WDYT?

"test browser promise":

```suggestion
		gotoPage := tb.promise(p.Goto(tb.URL("/get"), nil))
```

vs. "test browser wrap promise":


```suggestion
		gotoPage := tb.wrapPromise(p.Goto(tb.URL("/get"), nil))
```

vs. "test browser test promise"

```suggestion
		gotoPage := tb.testPromise(p.Goto(tb.URL("/get"), nil))
```